### ISSUE TITLE
Matrix unexpected rendering in DOC output

### ISSUE THREAD

**Issue by manuelfuenmayor** [plurimath/plurimath#164]
*In relation to https://github.com/metanorma/pdfa-iso-32000-2/issues/5 and https://github.com/metanorma/metanorma-standoc/issues/791*

```adoc
[stem%unnumbered]
++++
[(a,b,0),(c,d,0),(e,f,1)]
++++
````

Generated DOC:
![issue13-generated-doc](https://github.com/metanorma/metanorma-standoc/assets/33627611/1a31a031-b193-4835-a3d9-18241cc085d9)

Original:
![issue13-original](https://github.com/metanorma/metanorma-standoc/assets/33627611/6288c770-fa92-4781-ac43-195b768c75ff)

**Comment by suleman-uzair** [comment]
@manuelfuenmayor Using the latest version of **Plurimath 0.3.5**, I was able to generate the **MathML**(given below) of the example "[(a,b,0),(c,d,0),(e,f,1)]".
```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mstyle displaystyle="true">
    <mrow>
      <mo>[</mo>
      <mtable>
        <mtr>
          <mtd>
            <mi>a</mi>
          </mtd>
          <mtd>
            <mi>b</mi>
          </mtd>
          <mtd>
            <mn>0</mn>
          </mtd>
        </mtr>
        <mtr>
          <mtd>
            <mi>c</mi>
          </mtd>
          <mtd>
            <mi>d</mi>
          </mtd>
          <mtd>
            <mn>0</mn>
          </mtd>
        </mtr>
        <mtr>
          <mtd>
            <mi>e</mi>
          </mtd>
          <mtd>
            <mi>f</mi>
          </mtd>
          <mtd>
            <mn>1</mn>
          </mtd>
        </mtr>
      </mtable>
      <mo>]</mo>
    </mrow>
  </mstyle>
</math>
```
And I tested it in [**#demo**](https://www.mathjax.org/#demo), and it's generating the expected output.
Can you verify the **Plurimath** version, if the issue still persists, providing the generated **MathML** would be helpful.

**Comment by manuelfuenmayor** [comment]
Hi @suleman-uzair,
Yes, I'm using the same version of Plurimath (0.3.5).
This is the generated MathML from my side:
````
<formula id="_41e1f7c8-0fcb-b3e1-472b-985bbc3204e3" unnumbered="true"><stem type="MathML"><math xmlns="http://www.w3.org/1998/Math/MathML">
  <mstyle displaystyle="true">
    <mrow>
      <mo>[</mo>
      <mtable>
        <mtr>
          <mtd>
            <mi>a</mi>
          </mtd>
          <mtd>
            <mi>b</mi>
          </mtd>
          <mtd>
            <mn>0</mn>
          </mtd>
        </mtr>
        <mtr>
          <mtd>
            <mi>c</mi>
          </mtd>
          <mtd>
            <mi>d</mi>
          </mtd>
          <mtd>
            <mn>0</mn>
          </mtd>
        </mtr>
        <mtr>
          <mtd>
            <mi>e</mi>
          </mtd>
          <mtd>
            <mi>f</mi>
          </mtd>
          <mtd>
            <mn>1</mn>
          </mtd>
        </mtr>
      </mtable>
      <mo>]</mo>
    </mrow>
  </mstyle>
</math><asciimath>[(a,b,0),(c,d,0),(e,f,1)]</asciimath></stem></formula>
````

**Comment by manuelfuenmayor** [comment]
Perhaps this is not an issue from Plurimath?

**Comment by ronaldtse** [comment]
@suleman-uzair this is not a MathML issue but an OMML issue. Word only supports OMML not MathML. I suspect the output of OMML is wrong.

**Comment by ronaldtse** [comment]
@suleman-uzair how's this going?

**Comment by opoudjis** [comment]
No idea what was done here, but the matrix is continuing to crash Word.

This is the input MathML:

````
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mstyle displaystyle="true">
    <mrow>
      <mo>[</mo>
      <mtable>
        <mtr>
          <mtd>
            <mi>a</mi>
          </mtd>
          <mtd>
            <mi>b</mi>
          </mtd>
          <mtd>
            <mn>0</mn>
          </mtd>
        </mtr>
        <mtr>
          <mtd>
            <mi>c</mi>
          </mtd>
          <mtd>
            <mi>d</mi>
          </mtd>
          <mtd>
            <mn>0</mn>
          </mtd>
        </mtr>
        <mtr>
          <mtd>
            <mi>e</mi>
          </mtd>
          <mtd>
            <mi>f</mi>
          </mtd>
          <mtd>
            <mn>1</mn>
          </mtd>
        </mtr>
      </mtable>
      <mo>]</mo>
    </mrow>
  </mstyle>
</math>
````

This is the OMML that is crashing Word:

````
<m:oMath>
    <m:d><m:dPr><m:begChr m:val="["></m:begChr><m:endChr m:val="]"></m:endChr><m:sepChr m:val=""></m:sepChr><m:grow></m:grow></m:dPr><m:e><m:m><m:mpr><m:mcs><m:mc><m:mcPr><m:count m:val="3"></m:count><m:mcJc m:val="center"></m:mcJc></m:mcPr></m:mc></m:mcs><m:ctrlPr><w:rPr><w:rFonts w:ascii="Cambria Math" w:hAnsi="Cambria Math"></w:rFonts><w:i></w:i></w:rPr></m:ctrlPr></m:mpr><m:mr><m:e><m:r><m:t>a</m:t></m:r></m:e><m:e><m:r><m:t>b</m:t></m:r></m:e><m:e><m:r><m:t>0</m:t></m:r></m:e></m:mr><m:mr><m:e><m:r><m:t>c</m:t></m:r></m:e><m:e><m:r><m:t>d</m:t></m:r></m:e><m:e><m:r><m:t>0</m:t></m:r></m:e></m:mr><m:mr><m:e><m:r><m:t>e</m:t></m:r></m:e><m:e><m:r><m:t>f</m:t></m:r></m:e><m:e><m:r><m:t>1</m:t></m:r></m:e></m:mr></m:m></m:e></m:d>
  </m:oMath>
````

When I enter the matrix from scratch in Word, this is what I get:

````
<m:oMathPara><m:oMath><m:d><m:dPr><m:begChr
     m:val="["/><m:endChr m:val="]"/><span style='font-family:"Cambria Math",serif;
    mso-ascii-font-family:"Cambria Math";mso-hansi-font-family:"Cambria Math";
    font-style:italic;mso-bidi-font-style:normal'><m:ctrlPr></m:ctrlPr></span></m:dPr><m:e><m:m><m:mPr><m:mcs><m:mc><m:mcPr><m:count
          m:val="3"/><m:mcJc m:val="center"/></m:mcPr></m:mc></m:mcs><span
      style='font-family:"Cambria Math",serif;mso-ascii-font-family:"Cambria Math";
      mso-hansi-font-family:"Cambria Math";font-style:italic;mso-bidi-font-style:
      normal'><m:ctrlPr></m:ctrlPr></span></m:mPr><m:mr><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>a</m:r></span></i></m:e><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>b</m:r></span></i></m:e><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>0</m:r></span></i></m:e></m:mr><m:mr><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>c</m:r></span></i></m:e><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>d</m:r></span></i></m:e><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>0</m:r></span></i></m:e></m:mr><m:mr><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>e</m:r></span></i></m:e><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>f</m:r></span></i></m:e><m:e><i
       style='mso-bidi-font-style:normal'><span style='font-family:"Cambria Math",serif'><m:r>1</m:r></span></i></m:e></m:mr></m:m></m:e></m:d></m:oMath></m:oMathPara>
````

**Comment by opoudjis** [comment]
Problem found:

@suleman-uzair You are generating `m:mpr`. That is an error, it should be `m:mPr`.

**Comment by suleman-uzair** [comment]
@opoudjis 
I just released version **0.7.1**.

**Comment by opoudjis** [comment]
Thank you, confirmed and closed parent issue.
